index.js

```diff
@@ -1071,17 +1122,17 @@ if (check_dependencies()) {
           if (err instanceof Error) {
             if (err instanceof EthicalAdsWarning) {
               // Report these at a lower log level
-              console.debug(err.message);
+              logger.warn(err.message);
```

Comment:
I think we want these at warning. The main things that are displayed from this message are when there are no ads to show and when the ad request is blocked (ad blocker) or invalid (wrong publisher account). I think we want these to be visible at default verbosity.

Reply:
Yea, makes sense 👍